frontend/public/components/edit-yaml.jsx

```diff
@@ -452,7 +452,7 @@ export const EditYAML_ = connect(stateToProps)(
             </div>
           )}
 
-          <div className="pf-c-form co-m-page__body" style={{ display: 'flex', flexGrow: 1 }}>
+          <div className="pf-c-form co-m-page__body">
```

Comment:
The inline styles are not needed as [`.co-m-page__body`](https://github.com/openshift/console/blob/master/frontend/public/style/_layout.scss#L27-L28) already has them.